### PR TITLE
add files for cloud delpoyment

### PR DESCRIPTION
**Pull request by Aurora1024**
copy and modify files from lab-bluemix-cf

### CODE REVIEW

**Comment by mandy-cmd** [comment]
Just to confirm, this is to add deployment files but the cloud application hasn't been created right? I don't see anything on the cloud in your account yet. Or am I missing something?

**Review by vg2134 — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,151 @@
...
+
+  ######################################################################
+  # Add CouchDB docker container
+  ######################################################################
+  # docker run -d --name couchdb -p 5984:5984 -e COUCHDB_USER=admin -e COUCHDB_PASSWORD=pass couchdb
+  config.vm.provision "docker" do |d|
+    d.pull_images "couchdb"
+    d.run "couchdb",
+      args: "--restart=always -d --name couchdb -p 5984:5984 -v couchdb:/opt/couchdb/data -e COUCHDB_USER=admin -e COUCHDB_PASSWORD=pass"
+  end
```
Comment (mandy-cmd) [inline Dockerfile:116]: We are not using CouchDB but Postgres. I think a better way is to create the application first in the cloud and test if you could deploy to cloud with the deployment files.

**Comment by Aurora1024** [comment]
Can anyone please help look at the branch and try push to cloud? 
I stuck here. The ibm always look for port 8080 even after I change procfile to 5000
I also post the question in trouble shooting channel
After login to your ibm account 
run: 
`ibmcloud target -o kw2669@nyu.edu -s dev`

       if use your own email please add  ElephantSQL by:
      ic cf cups ElephantSQL -p '{"url":"postgres://admin:passw0rd@db.elephantsql.com:5432/testdb"}
finally 
`ic cf push`

**Comment by mandy-cmd** [comment]
Fixed deployment to cloud in #eb583. Should be good now. Approving and merging.

**Pull request merged**